specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/examples/PrivateClouds_Get.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "api-version": "2019-08-09-preview",
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "group1",
+    "privateCloudName": "cloud1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscription-id}/resourceGroups/group1/providers/Microsoft.VMwareVirtustream/privateClouds/cloud1",
+        "location": "eastus2",
+        "name": "cloud1",
+        "properties": {
+          "circuit": {
+            "authorizations": [],
+            "expressRouteID": "/subscriptions/{subscription-id}/resourceGroups/tnt13-41a90db2-9d5e-4bd5-a77a-5ce7b58213d6-eastus2/providers/Microsoft.Network/expressroutecircuits/tnt13-41a90db2-9d5e-4bd5-a77a-5ce7b58213d6-eastus2-xconnect",
+            "primarySubnet": "192.168.53.0/30",
+            "secondarySubnet": "192.168.53.4/30",
+            "expressRoutePrivatePeeringID": "/subscriptions/{subscription-id}/resourceGroups/tnt42-cust-p01-dmo01/providers/Microsoft.Network/expressroutecircuits/tnt42-cust-p01-dmo01-er/peerings/AzurePrivatePeering"
+          },
+          "cluster": {
+            "clusterId": 1,
+            "clusterSize": 4,
+            "hosts": [
+              "fakehost18.nyc1.kubernetes.center",
+              "fakehost19.nyc1.kubernetes.center",
+              "fakehost20.nyc1.kubernetes.center",
+              "fakehost21.nyc1.kubernetes.center"
+            ]
+          },
+          "clusters": [],
```

Comment:
>clusters [](start = 11, length = 8)

This looks like it shows the same information as the nested resource type. Is it read only, or do you also allow creation of clusters via the `clusters` property on the private cloud?

Reply:
It is read only. It does not allow creation of clusters via that property.

Reply:
Thanks for confirming.